docs/quickstart-gke.md

```diff
@@ -84,6 +84,7 @@ In this section, you will deploy a Google Kubernetes Engine (GKE) cluster using
       + https://prometheus.${BKPR_DNS_ZONE}/oauth2/callback
       + https://kibana.${BKPR_DNS_ZONE}/oauth2/callback
       + https://grafana.${BKPR_DNS_ZONE}/oauth2/callback
+      + https://auth.${BKPR_DNS_ZONE}/oauth2/callback
```

Comment:
Why don't you remove the three previous lines, as they won't be needed anymore IIUC?

Reply:
It was too hard to explain that the old URLs are only needed for releases before this PR, and that in the future (after this PR) they will need to go back and change the URLs (we can't do this automatically because Google has no API for oauth2 setup).  I couldn't come up with a solution that was any better than "just list all of them" - if we had a system of versioned docs then there might have been other options available.

Note: We can remove the old URLs after we have made a release containing this PR, but again the cost of leaving them is very low.